pkg/client/restclient/config.go

```diff
@@ -45,11 +45,11 @@ const (
 // Config holds the common attributes that can be passed to a Kubernetes client on
 // initialization.
 type Config struct {
-	// Host must be a host string, a host:port pair, or a URL to the base of the apiserver.
-	// If a URL is given then the (optional) Path of that URL represents a prefix that must
-	// be appended to all request URIs used to access the apiserver. This allows a frontend
-	// proxy to easily relocate all of the apiserver endpoints.
-	Host string
+	// Hosts must be a slice of host strings, host:port pairs, or a URLs to the base of the
+	// apiservers. If a URL is given then the (optional) Path of that URL represents a prefix
+	// that must be appended to all request URIs used to access the apiserver. This allows a
+	// frontend proxy to easily relocate all of the apiserver endpoints.
+	Hosts []string
```

Comment:
> all those api-servers are equal and which one i, as an operator, should select, what it will affect

Even with this PR that isn't the case - the operator may very well know better than you which one they want to use primarily.  I do think some of the changes below are cleaner, but this is a part of our API and will have a ripple impact on our clients.

I think we will have to make changes to kubeconfig, but `alternateHosts` would at least be additive, and old kubeconfigs would continue to work.  If we end ups with `alternateHosts` in the kubeconfig, then we might as well have them internally.